lib/jbuilder/jbuilder_template.rb

```diff
@@ -55,7 +55,9 @@ def partial!(*args)
     if args.one? && _is_active_model?(args.first)
       _render_active_model_partial args.first
     else
-      _render_explicit_partial(*args)
+      options = args.extract_options!.dup
+      options[:partial] = args.first if args.present?
+      _render_partial_with_options options
```

Comment:
Was able to refactor this in such a way that we no longer require `_render_explicit_partial`. That method was actually _already_ mutating options, either directly via `as = locals.delete(:as)`, or later on when it called `_render_partial_with_options`.

This new setup prevents that from happening and also saves on a bunch of work in `_render_explicit_partial`, which was basically doing things that were already handled in `_render_partial_with_options`.